lib/tic_tac_toe_web/controllers/game_controller.ex

```diff
@@ -37,11 +38,34 @@ defmodule TicTacToeWeb.GameController do
         end
     end
 
+    def validate_move(current_board, incoming_move) do
+        parsed_move = to_string(incoming_move)
+
+        case (Board.valid_move?(current_board, parsed_move)) do
```

Comment:
Is an if more idiomatic than case since true/false are the only options?

Reply:
Thanks I changed this to an if/else.